apps/src/templates/sectionProgress/SectionProgressToggle.jsx

```diff
@@ -27,62 +28,48 @@ class SectionProgressToggle extends React.Component {
   };
 
   state = {
+    initialView: ViewType.SUMMARY,
```

Comment:
should this be called currentView instead of initialView?

Reply:
also... why is its value set to ViewType.SUMMARY instead of this.props.currentView?

Reply:
if `this.props.currentView` is the right initial value, then it looks like you can consolidate this with `selectedToggle`.

Reply:
I need to store two pieces of information in order to set `old_view` and `new_view` correctly in the logging.  I need to know the tab we came from and the tab we're viewing.  I'm using `initialView` to track where we came from and `currentView` to track where we are now, because it's what the view was initially and what it is currently now. Does that make sense?  Any thoughts on clearer names?

I set initialView to summary because that's the default view when you render the page, unless it's set in `componentWillReceiveProps(nextProps)` so I updated initialView to this.props.currentView there.